plugins/reports/api/reports.js

```diff
@@ -102,25 +103,30 @@ var metrics = {
          * @param {func} cb - callback function
          */
         function processUniverse(cb) {
-            if (versionInfo.title.indexOf("Countly") > -1) {
-                var options = {
-                    uri: 'http://count.ly/email-news.txt',
-                    method: 'GET',
-                    timeout: 1000
-                };
+            if (!offlineMode) {
+                if (versionInfo.title.indexOf("Countly") > -1) {
+                    var options = {
+                        uri: 'http://count.ly/email-news.txt',
+                        method: 'GET',
+                        timeout: 1000
+                    };
 
-                request(options, function(error, response, body) {
-                    if (!error) {
-                        try {
-                            var arr = JSON.parse(body);
-                            report.universe = arr[Math.floor(Math.random() * arr.length)];
-                        }
-                        catch (ex) {
-                            console.log(ex);
+                    request(options, function(error, response, body) {
+                        if (!error) {
+                            try {
+                                var arr = JSON.parse(body);
+                                report.universe = arr[Math.floor(Math.random() * arr.length)];
+                            }
+                            catch (ex) {
+                                console.log(ex);
+                            }
                         }
-                    }
+                        cb(null);
+                    });
+                }
+                else {
                     cb(null);
-                });
+                }
```

Comment:
we still need to call the cb in else case here, or else process just stops and hangs